### ISSUE TITLE
Accessibility Violation: Multiple "search" landmarks with the same parent region are not distinguished from one another because they have the same "" label and Multiple elements with "search" role do not have unique labels

### ISSUE THREAD

**Issue by AmirHussain93** [carbon-design-system/carbon-components-angular#2512]
**1. Multiple "search" landmarks with the same parent region are not distinguished from one another because they have the same "" label
2. Multiple elements with "search" role do not have unique labels**

## Detailed description
This issue is specific to the `<div>` element which is used inside `<ibm-search>` and `<ibm-table-toolbar-search>` component

In my case I am using` <ibm-search>` at two places along with `<ibm-table-toolbar-search>` in a single page. Which is resulting in this accessibility violation.

**Using ARIA landmarks to identify regions of a page**
The purpose of this technique is to provide programmatic access to sections of a web page. Landmark roles (or "landmarks") programmatically identify sections of a page. Landmarks help assistive technology (AT) users orient themselves to a page and help them navigate easily to various sections of a page.

There are cases when a particular landmark role could be used more than once on a page, such as on primary and secondary navigation menus. In these cases, identical roles should be disambiguated from each other using a valid technique for labelling regions.

Landmarks should supplement native semantic markup such as HTML headings, lists and other structural markup. Landmarks are interpretable by WAI-ARIA-aware assistive technologies and are not exposed by browsers directly to users (see examples below).

It is a best practice to include ALL content on the page in landmarks, so that screen reader users who rely on them to navigate from section to section do not lose track of content.

**Example 1**
The following example shows a best practice of how landmarks might be added to an HTML4 or XHTML 1.0 document in situations where there are more than two of the same type of landmark on the same page. For instance, if a navigation role is used multiple times on a Web page, each instance may have a unique label specified using aria-labelledby:

```
<div id="leftnav" role="navigaton" aria-labelledby="leftnavheading">
<div id="rightnav" role="navigation" aria-labelledby="rightnavheading">
```

**Example 2**
The following example shows a best practice of how landmarks might be added to an HTML4 or XHTML 1.0 document in situations where there are more than two of the same type of landmark on the same page, and there is no existing text on the page that can be referenced as the label.

```
<div id="leftnav" role="navigaton" aria-label="Primary">
<div id="rightnav" role="navigation" aria-label="Secondary">
```

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug

> Is this issue related to a specific component?
yes, `<ibm-search>`

> What did you expect to happen? What happened instead? What would you like to see changed?
Multiple elements with "search" should have unique labels. There should be an option to provide label to `<ibm-search>`

> What browser are you working in?
chrome Version 112.0.5615.49 (Official Build) (arm64)

> What version of the Carbon Design System are you using?
carbon-components@10.56.0
carbon-components-angular@4.72.1

## Additional information

* Screenshots or code
<img width="1724" alt="image" src="https://user-images.githubusercontent.com/27803197/230595372-be5b647c-3b9f-4134-bc13-d07e728ab463.png">

**Comment by AmirHussain93** [comment]
@Akshat55 Any update on this issue ?

**Comment by Akshat55** [comment]
Will look into this! @AmirHussain93

**Comment by Akshat55** [comment]
> 1. Multiple "search" landmarks with the same parent region are not distinguished from one another because they have the same "" label
> 2. Multiple elements with "search" role do not have unique labels

@AmirHussain93 - Unique labels are missing. If you have multiple searches on the page, you need to let the screen reader know what those searches are for.

The labels are hidden, but available to the screen reader. Please add unique labels to each search and your issue will be resolved. 

```html
		<!-- Add label attribute WITH unique value representing what the search is for. -->
		<ibm-search label="Label!"></ibm-search>
```

Closing this issue as the fix is required on your end, feel free to respond or reopen.
